stdlib/public/Distributed/DistributedActorSystem.swift

```diff
@@ -326,6 +323,52 @@ extension DistributedActorSystem {
   }
 }
 
+/// Represents a 'target' of a distributed call, such as a `distributed func` or
+/// `distributed` computed property. Identification schemes may vary between
+/// systems, and are subject to evolution.
+///
+/// Actor systems generally should treat the `identifier` as an opaque string,
+/// and pass it along to the remote system for in their `remoteCall`
+/// implementation. Alternative approaches are possible, where the identifiers
+/// are either compressed, cached, or represented in other ways, as long as the
+/// recipient system is able to determine which target was intended to be
+/// invoked.
+///
+/// The string representation will attempt to pretty print the target identifier,
+/// however its exact format is not specified and may change in future versions.
+@available(SwiftStdlib 5.7, *)
+public struct RemoteCallTarget: CustomStringConvertible {
```

Comment:
I was quite torn on the representation of this to be honest.

We could internally store:

```
  private enum _Storage {
     case mangledName(String)
   }
```

but then the initializer would be mangledName and cause people to over worry about it again...

Internal representation we can always change in the future right @DougGregor ? The struct is not frozen.